{{cookiecutter.package_name}}/.github/workflows/build.yml

```diff
@@ -74,11 +74,11 @@ jobs:
       - name: Install dependencies
         run: |
           pip install .[test]
-      - name: Test
-        run: |
-          pytest ./tests
       - name: Lint
         run: |
-          pip install black==22.1.0 isort
+          pip install black==22.3.0 isort
```

Comment:
This choice predates this PR, but it's curious that we pin `black` here but not `isort`...